test/reporters/src/data.ts

```diff
@@ -174,4 +174,60 @@ const tasks: Task[] = [
 file.tasks = [suite]
 suite.tasks = tasks
 
-export const files = [file]
+const files = [file]
+
+function createSuiteHavingFailedTestWithXmlInError(): File[] {
```

Comment:
To make test cleaner you can move this to a separate file.

Reply:
Moved to `data-for-junit.ts`